ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Settings.hs

```diff
@@ -0,0 +1,336 @@
+module Ariadne.UI.Qt.Widgets.Settings
+    ( Settings
+    , initSettings
+    , showSettingsWindow
+    ) where
+
+import Control.Lens (makeLensesWith)
+import Data.Bits
+import Web.Browser (openBrowser)
+
+import qualified Graphics.UI.Qtah.Widgets.QDialog as QDialog
+import Graphics.UI.Qtah.Signal (connect_)
+import Graphics.UI.Qtah.Core.Types (alignHCenter, alignVCenter)
+import Graphics.UI.Qtah.Widgets.QSizePolicy (QSizePolicyPolicy(..))
+
+import qualified Graphics.UI.Qtah.Core.QObject as QObject
```

Comment:
Sort imports. Or just set up your editor to run `stylish-haskell` with our rules (`.stylish-haskell.yaml` in repo root).